lib/Extension/WorseReflectionExtra/Application/OffsetInfo.php

```diff
@@ -47,8 +48,8 @@ public function infoForOffset(string $sourcePath, int $offset, $showFrame = fals
             $frame = [];
 
             foreach (['locals', 'properties'] as $assignmentType) {
-                /** @var $local Variable */
                 foreach ($result->frame()->$assignmentType() as $local) {
+                    /** @var Variable $local */
```

Comment:
this should also be generic, but not sure if PHPStan is smart enough here, Phpactor probably is not.